### DISCUSSION TITLE
How to use multiple Wine builds?

### DISCUSSION THREAD

**Discussion by epassaro** [tkashkin/GameHub#591] [Q&A] [answered]
Hi!

I'm currently using Wine staging 5.x downloaded from [Kron4ek/Wine-Builds](https://github.com/Kron4ek/Wine-Builds)  and extracted to `/opt`. Then, I edited `~/.profile` to extend the `PATH` variable. 

This method works flawlessly. But, is there any way to make GameHub detect different Wine versions (like Proton)? I understand GameHub detects the Wine available on the system, like the output of `which wine`.

**Comment by ghost** [comment]


**Reply by tkashkin** [reply]
@maklamj You could at least do a bit of research before stating incorrect information:

* @Lucki's [`epic`](https://github.com/Lucki/GameHub/tree/epic) branch only contains partial EGS implementation.
* This branch is based on my [`refactoring`](https://github.com/tkashkin/GameHub/tree/refactoring) branch - the branch that actually implements the features you're talking about. It is WIP and not all features work in the `refactoring` branch - that's why it's not merged into `dev` or `master` yet.
* @Lucki's PR (#538) is not merged because it is also a WIP draft and it is not tested.

>  it wasn't merged for no reason (abandonware, lol)

Sure, call it whatever you want. I have a life and a job, I don't have multiple hours a day to dedicate to a project I gain absolutely nothing from. I continue development when I have free time and motivation to do it.

**Comment by tkashkin** [comment, marked as answer]
It is implemented in the `refactoring` branch. It's WIP and some features are not implemented there. See #459 for more information.
